### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Poussinou**
Hi,

This small and useless PR adds two buttons to get your app on F-Droid and the Play Store, Enjoy :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,11 @@ Everything else is under the AGPL3 (see [LICENSE](LICENSE))
 
 [Try beta versions at DeployGate](https://deploygate.com/distributions/fc678589a532654d2f7c925f91c896c8c7c3e3db)
 
+<a href="https://f-droid.org/packages/net.tevp.postcode/" target="_blank">
+<img src="https://f-droid.org/badge/get-it-on.png" alt="Get it on F-Droid" height="90"/></a>
+<a href="https://play.google.com/store/apps/details?id=net.tevp.postcode" target="_blank">
+<img src="https://play.google.com/intl/en_us/badges/images/generic/en-play-badge.png" alt="Get it on Google Play" height="90"/></a>
```
Comment (palfrey) [inline README.md]: This should be redone as Markdown equivalents. Yes, HTML can be embedded in Markdown, but it's untidy. e.g. see https://gitlab.com/fdroid/artwork/blob/master/badge/README.md for a guide on the correct markup for F-Droid

Comment (Poussinou) [reply README.md]: Interesting ressource, thanks for showing this link to me !

File: README.md [hunk 2]
```diff
@@ -10,6 +10,13 @@ Everything else is under the AGPL3 (see [LICENSE](LICENSE))
 
 [Try beta versions at DeployGate](https://deploygate.com/distributions/fc678589a532654d2f7c925f91c896c8c7c3e3db)
 
+[<img src="https://f-droid.org/badge/get-it-on.png"
+      alt="Get it on F-Droid"
+      height="90">](https://f-droid.org/packages/net.tevp.postcode/)
+[<img src="https://play.google.com/intl/en_us/badges/images/generic/en-play-badge.png"
+      alt="Get it on F-Droid"
```
Comment (palfrey) [inline README.md]: 'Get it on F-Droid' needs rewriting :) Thanks for all these changes

Comment (Poussinou) [reply README.md]: Pfiou that was hard xD Sorry for the loss of time

Comment (palfrey) [reply README.md]: Not a problem. Always glad to see pull requests, especially small ones. They're much easier to review!

**Pull request merged**